src/backend/utils/elastic.js

```diff
@@ -111,10 +111,13 @@ const waitOnReady = async () => {
   });
 
   const connectivity = new Promise((resolve) => {
-    intervalId = setInterval(() => {
-      checkConnection()
-        .then(resolve)
-        .catch(() => logger.info('Attempting to connect to elasticsearch...'));
+    intervalId = setInterval(async () => {
```

Comment:
You can't mix timeout/interval callbacks with `async` functions (or Promises) like this.  I honestly can't remember what the "right" way is to do this, but there's a package that does it https://github.com/ealmansi/set-interval-async.

I'd need to research this more to give you a proper answer.